vmdb/app/models/orchestration_stack_openstack_infra.rb

```diff
@@ -0,0 +1,2 @@
+class OrchestrationStackOpenstackInfra < OrchestrationStack
+end
```

Comment:
Parent class `OrchestrationStack` default has some relations that limited to clouds.
`belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "EmsCloud"`
`has_many   :vms, :class_name => "VmCloud"`
We need to override these relations in this class. Similar changes are needed for other classes.